muni_portal/core/views/api/webpush.py

```diff
@@ -25,3 +27,15 @@ def create(self, request, *args, **kwargs):
         except Exception as e:
             logger.error(e)
             return Response(status=status.HTTP_400_BAD_REQUEST, data={"message": e})
+
+
+class VapidApiView(RetrieveAPIView):
+
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request, *args, **kwargs):
+        if not os.path.isfile(settings.VAPID_PUBLIC_KEY):
+            Response(status=status.HTTP_404_NOT_FOUND)
+        with open(settings.VAPID_PUBLIC_KEY, "r") as file:
```

Comment:
Remember we prefer using the value approach over the file path approach. This won't support me passing in the key as a base64 value. Please fix